test/context/directory/clientGrants.test.js

```diff
@@ -103,33 +120,65 @@ describe('#directory context clientGrants', () => {
   it('should dump client grants', async () => {
     const dir = path.join(testDataDir, 'directory', 'clientGrantsDump');
     cleanThenMkdir(dir);
-    const context = new Context({ AUTH0_INPUT_FILE: dir }, mockMgmtClient());
-
-    context.assets.clientGrants = [
-      { audience: 'https://test.myapp.com/api/v1', client_id: 'My M2M', scope: ['update:account'] },
-    ];
-
-    await handler.dump(context);
-    const clientGrantsFolder = path.join(dir, constants.CLIENTS_GRANTS_DIRECTORY);
-
-    expect(loadJSON(path.join(clientGrantsFolder, 'My M2M.json'))).to.deep.equal(
-      context.assets.clientGrants[0]
+    const context = new Context(
+      { AUTH0_INPUT_FILE: dir },
+      {
+        ...mockMgmtClient(),
+        resourceServers: {
+          getAll: () => [
+            {
+              id: 'resource-server-1',
+              name: 'Payments Service',
+              identifier: 'https://payments.travel0.com/api',
+            },
+            {
+              id: 'resource-server-2',
+              name: 'Auth0 Management API',
+              identifier: 'https://travel0.us.auth0.com/api/v2',
+            },
+          ],
+        },
+      }
     );
-  });
-
-  it('should dump client grants sanitized', async () => {
```

Comment:
This test case was introduced with #92 but is functionally identical to the `'should dump client grants'` test so I'm removing it.